Sources/TuistGenerator/Generator.swift

```diff
@@ -1,62 +1,108 @@
 import Basic
+import Foundation
 import TuistCore
 
+public struct GeneratorConfig {
```

Comment:
This is effectively a move from `TuistKit/Generator/Generator.swift` > `TuistGenerator/Generator.swift`

`TuistKit/Generator/Generator.swift` only now hosts the extension that deals with the manifest - we could beef it up to its own class too